src/cmd/test.ts

```diff
@@ -14,11 +14,15 @@ import { validateValues } from './validate-values'
 
 const cmdName = getFilename(import.meta.url)
 const tmpFile = '/tmp/otomi/test.yaml'
-const debug: OtomiDebugger = terminal(cmdName)
+let debug: OtomiDebugger = terminal(cmdName)
 
 const cleanup = (argv: Arguments): void => {
+  debug = terminal(cmdName)
```

Comment:
Why do we need to set it here? It should be set up in the `setup` function. And only needs to be defined once.

Reply:
Ok sounds good.